frontend/src/components/MeetingRoom/DeviceSettingsMenu/DeviceSettingsMenu.spec.tsx

```diff
@@ -120,7 +123,9 @@ describe('DeviceSettingsMenu Component', () => {
       await act(() => (outputDevicesElement.children[2] as HTMLOptionElement).click?.());
 
       expect(mockSetAudioOutputDevice).toHaveBeenCalledWith(audioOutputDevices[2].deviceId);
-      await expect((outputDevicesElement.children[2] as HTMLOptionElement).selected).toBe(true);
+      expect((outputDevicesElement.children[2] as Element).classList.contains('Mui-selected')).toBe(
```

Comment:
Is it possible to check the visual indicator we use is present for this output device? I'm not a big fan of checking the presence of a class.

Reply:
I don't see a particular issue with checking for a class. I will add an extra expect to check that the following element is not currently selected, though, to be extra certain